BD-3380: Add exception events for exit criteria (#7979)
braze-inc/braze-docs: _docs/_user_guide/engagement_tools/canvas/create_a_canvas/exit_criteria.md

You can also include segments and filters in the exit criteria, meaning users who match the segment or filter will exit the Canvas and will not receive any further messaging. If the first step in a Canvas is a Delay step with a five-day delay, then the exit criteria will apply at the end of this step. So, if a user meets the exit criteria, they will exit at the end of the five days.

{% alert note %}
Array attributes aren’t currently supported as exit criteria on exception events.
{% endalert %}

## Exception events

Additional exception events include:
- Making a purchase
- Starting a session
- Performing a custom event
- Performing a conversion event
- Adding an email address
- Changing a custom attribute value
- Updating a subscription status
- Updating a subscription group status
- Interacting with a campaign
- Entering a location
- Triggering a geofence
- Sending an SMS inbound message
- Sending a WhatsApp inbound message
- Sending a LINE inbound message
- Performing a cart updated event
- Performing a checkout completed event
- Performing a checkout started event

## Use case

> Let's say you want to target users who haven't made any purchases at your company yet. You would select **Make Purchase** as the exception event. Next, select **Add Trigger**. When your Canvas launches, your audience now excludes users who have made any purchases with the following **Exit Criteria** settings. 

Reviewer: ```suggestion
Let's say you want to target users who haven't made any purchases at your company yet. First, select **Make Purchase** as the exception event. Next, select **Add Trigger**. When your Canvas launches, your audience will exclude users who have made any purchases with the following **Exit Criteria** settings. 
```
